logbot.py

```diff
@@ -66,10 +66,16 @@
 
 #urlfinder = re.compile("(?i)\b((?:[a-z][\w-]+:(?:/{1,3}|[a-z0-9%])|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}/)(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'\".,<>?«»“”‘’]))")
 
-flaskapp = Flask(__name__)
-flaskapp.config["TEMPLATES_AUTO_RELOAD"] = True
+# We use a blueprint so that we can have a prefix in the URL
+flaskapp = Blueprint('logbot-blueprint', __name__)
 
-@flaskapp.template_filter('md5')
+app = Flask(__name__)
+app.config["TEMPLATES_AUTO_RELOAD"] = True
+
+# The URL prefix on the server
+app.config["APPLICATION_ROOT"] = "/irclogs"
```

Comment:
Instead of hardcoding this, could you use something like `app.config["APPLICATION_ROOT"] = os.getenv("APPLICATION_ROOT", "/irclogs")`?

Reply:
Haha, I actually thought about that and just hoped nobody would comment on it! Will fix :)